covasim/webapp/index.html

```diff
@@ -92,6 +93,7 @@ <h5 class="modal-title" id="exampleModalLabel">About Covasim</h5>
             <div :id="key+'-wpr'">
               <label :for="key">{{par.name}}:</label>
               <input :id="key" :disabled="running" class="form-control" v-model="par.best">
+              <span aria-live="assertive" v-if="paramError[key]" class="text-danger form-error">{{paramError[key]}}</span>
```

Comment:
Not sure if there is any need for screen reader support yet but technically, [aria-live=assertive] needs to be in the DOM when the page first loads and dynamic content (v-if elements) should be children of aria-live elements in order for screen readers to pick up the changes. Ran into this scenario myself quite a few times